packages/suite/src/support/messages.ts

```diff
@@ -3038,7 +3038,7 @@ const definedMessages = defineMessages({
     TR_SEED_CHECK_SUCCESS_DESC: {
         id: 'TR_SEED_CHECK_SUCCESS_DESC',
         defaultMessage:
-            'Your seed is valid and has just been successfully checked. Please take great care of it and/or hide it back where you are goint to find it.',
+            'Your seed is valid and has just been successfully checked. Please take great care of it and/or hide it back where you are going to find it.',
```

Comment:
this one is fine as there is no copy from crowdin, this default message is really used in the app